packages/components/src/Offer/Form/Checkout.tsx

```diff
@@ -151,6 +148,20 @@ const OfferFormCheckout: FC<Props> = ({
                 placeholder={t('offer.form.checkout.quantity.placeholder')}
                 {...register('quantity', {
                   required: t('offer.form.checkout.validation.required'),
+                  validate: (value) => {
+                    if (
+                      parseInt(value, 10) < 1 ||
+                      parseInt(value, 10) >
+                        parseInt(offer.availableQuantity, 10)
+                    ) {
+                      return t('offer.form.checkout.validation.in-range', {
+                        available: parseInt(offer.availableQuantity, 10),
```

Comment:
Seems the translation wants `quantityAvailable`:
```suggestion
                        quantityAvailable: parseInt(offer.availableQuantity, 10),
```

Reply:
normalized all translations